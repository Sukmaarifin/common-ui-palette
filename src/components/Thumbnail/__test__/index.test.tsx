import React from 'react';
import { mount } from 'enzyme';
import toJSON from 'enzyme-to-json';

import Thumbnail, { ThumbnailProps } from '../Thumbnail';

describe('Thumbnail test', () => {
  const props: ThumbnailProps = {
    imageSource: '#some-cool-image.jpg',
    altText: 'Indonesia',
    size: 'small',
  };

  // snapshot
  it('renders correctly', () => {
    const wrapper = mount(<Thumbnail {...props} />);

    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
