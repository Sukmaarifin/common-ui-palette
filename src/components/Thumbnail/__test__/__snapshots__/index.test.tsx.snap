// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Thumbnail test renders correctly 1`] = `
<ThumbnailComponent
  altText="Indonesia"
  imageSource="#some-cool-image.jpg"
  size="small"
>
  <img
    alt="Indonesia"
    className="makeStyles-image-2 makeStyles-small-3"
    onError={[Function]}
    src="#some-cool-image.jpg"
  />
</ThumbnailComponent>
`;
